{{ cookiecutter.folder_name }}/.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-exclude = .git,__pycache__,build,dist,{{ cookiecutter.import_name }}/_version.py,docs/source/conf.py
+exclude = .git,__pycache__,build,dist,{{ cookiecutter.import_name }}/_version.py
 max-line-length = 88
 select = C,E,F,W,B,B950
-extend-ignore = E203, E501
+extend-ignore = E203, E501, E226, W503, W504
```

Comment:
These are:

Error | Description
-- | --
E203 | Whitespace before ':'
E501 | Line too long (82 > 79 characters)
E226 | Missing whitespace around arithmetic operator
W503 | Line break occurred before a binary operator
W504 | Line break occurred after a binary operator

And B950:

> B950: Line too long. This is a pragmatic equivalent of pycodestyle's E501: it considers "max-line-length" but only triggers when the value has been exceeded by more than 10%. You will no longer be forced to reformat code due to the closing parenthesis being one character too far to satisfy the linter. At the same time, if you do significantly violate the line length, you will receive a message that states what the actual limit is. This is inspired by Raymond Hettinger's ["Beyond PEP 8" talk](https://www.youtube.com/watch?v=wf-BqAjZb8M) and highway patrol not stopping you if you drive < 5mph too fast. Disable E501 to avoid duplicate warnings. Like E501, this error ignores long shebangs on the first line and urls or paths that are on their own line:
> 
> ```
> #! long shebang ignored
> 
> # https://some-super-long-domain-name.com/with/some/very/long/paths
> url = (
>     "https://some-super-long-domain-name.com/with/some/very/long/paths"
> )
> ```


E203 surprised me a bit - I didn't recall that being in there. Perhaps there was a reason?

Reply:
E203 is from `black`'s recommendations: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html